appveyor.yml

```diff
@@ -1,60 +1,65 @@
-image: Visual Studio 2019
+# Source: https://github.com/krlmlr/r-appveyor/blob/master/appveyor.yml
 
-version: '{build}'
+# DO NOT CHANGE the "init" and "install" sections below
 
 # Download script file from GitHub
 init:
-  - ps: Update-AppveyorBuild -Version "$($env:ospsuite_version).$($env:appveyor_build_version)"
-  - ps: |
```

Comment:
This shuold stay. Why did you remove this?

Reply:
The versioning of _ospsuite.utils_ is quite different from that in rest of the _ospsuite_, right? 

Won't `$ospsuite_version` produce incorrect version?

Reply:
yes that's why I said this can go. (the versionning)
But the rest that was removed (Bootstrap etc..) needs to come back. This is part of the original script (unless the script has changeD)

Reply:
Cool. Added it back in.

Reply:
I had changed it to match the original script, which has changed in the meantime. But now I have reverted to the one we have been using.